examples/artemis-multicast-consumer/artemis_multicast_consumer.out

```diff
@@ -0,0 +1,8 @@
+# Make sure to have the ActiveMQ Artemis broker running
+
+# Navigate to the directory that contains the 'artemis_multicast_consumer.bal' file and issue the
+# 'ballerina run' command.
+$ ballerina run artemis_multicast_consumer.bal
+# The ActiveMQ Artemis consumer runs as a Ballerina service and listens to the subscribed queue
+
+Initiating service(s) in 'artemis_multicast_consumer.bal'
```

Comment:
```suggestion
Initiating service(s) in the 'artemis_multicast_consumer.bal' file
```

Reply:
This is what is printed in the terminal from the language side code, hence cannot be changed